src/patternfly/components/ContextSelector/context-selector.scss

```diff
@@ -118,6 +130,30 @@
     }
   }
 
+  &.pf-m-plain {
+    --pf-c-context-selector__toggle--PaddingRight: var(--pf-c-context-selector__toggle--m-plain--PaddingRight);
+    --pf-c-context-selector__toggle--PaddingLeft: var(--pf-c-context-selector__toggle--m-plain--PaddingLeft);
+
+    display: inline-block;
```

Comment:
This should only apply to the non-text plain toggle - can you move it below to the `&:not(.pf-m-text)` selector below? It's causing the toggle to lose its flex layout, so it looks like this currently 
<img width="643" alt="Screen Shot 2021-06-04 at 3 08 53 PM" src="https://user-images.githubusercontent.com/35148959/120857660-35bf2000-c547-11eb-947e-07de3477f295.png">